content/calls/main_track.md

```diff
@@ -4,189 +4,144 @@ date: 2023-08-09T08:26:07+02:00
 
-ICAPS 2024 furthermore follows AAAI policy in that any AI system, including Generative Models such as Chat-GPT, BARD, or DALL-E, does not satisfy the criteria for authorship of papers published by AAAI Press (ICAPS’ publisher) and, as such, also cannot be used as a citable source in papers published by AAAI press. Authors assume full responsibility for the content of submitted work, including checking for plagiarism and veracity of all text.
+Over-length papers will be rejected without review. A paper is over-length whenever content other than the ethical 
+impacts statement (see below) or acknowledgments section appears on page 9 (resp. Page 5) of long (resp. short) papers.
 
-## Best paper fast-track to top AI journals
+All papers that satisfy length and formatting requirements will be reviewed against standard criteria such as relevance, 
+originality, significance, clarity, and soundness. Submissions are expected to meet the high standards of publication 
+expected by ICAPS. Please refer to the 
+[conference](https://icaps25.icaps-conference.org/organization/roles_and_guidelines)
+[website](https://icaps25.icaps-conference.org/organization/conditions_for_acceptance) 
+for further details and reviewing guidelines. 
 
-Following the ICAPS tradition, in ICAPS 2024, we will give awards to the best paper and best student paper. These papers may also receive an invitation to submit an extended version to a top AI journal (e.g., the Journal of Artificial Intelligence Research (JAIR) or the Artificial Intelligence Journal (AIJ)).
+The review process has three possible outcomes: (1) acceptance for oral presentation and inclusion in the archival 
+proceedings of the ICAPS conference series; (2) acceptance for poster presentation and non-archival extended abstract, 
+for dissemination via the ICAPS 2025 website; (3) rejection.
 
-## Ethical/Societal Impact
+Papers submitted to ICAPS 2025 may not be submitted to other conferences or journals during the ICAPS 2025 review period, 
+nor may they be already under review, accepted, or published in other conferences or journals. 
 
-It is optional for authors to include a statement of the potential broader impact of their work, including its ethical aspects and future societal consequences. This statement can be included in either the main body pages or the reference page. If such a statement is not included in the paper, but the reviewers deem that such a statement is necessary, then the authors will be asked to provide one during the author response period for review. If the paper is accepted, the statement provided will need to be incorporated into the camera-ready version.
+Authors making multiple submissions must ensure that each submission has significant, unique content. Papers sharing 
+co-authors and substantial overlap will be rejected. In case of doubt, please contact the program chairs of ICAPS 2025 
+before submission.
```

Comment:
Well, that is part of the question we put forward to the Council. Do you have any thoughts on this?

Reply:
I'd add it in the CFP, just so it is stated and you won't get authors telling you later on that rules were not clear.

Reply:
@nirlipo I am confused because Github is diffing against 2024's CfP. There are some significant divergences in structure. We have the Ethical Statement in there already.

Reply:
Our question was to see if we had to add a clause that says "Or else these are the consequences: ..."

Reply:
> Should we add a statement about not allowing adding authors after the full paper deadline?

I missed this comment. We haven't configured OpenReview to allow this. The author list at submission is final and the submission forms are quite clear about this.